index.bs

```diff
@@ -340,8 +340,17 @@ Issue: Include cross references to the specs for these hash functions.
     value.  This signals to the listening agent that it should connect to the
     advertising agent to discover updated metadata.
 
-Issue: Add examples of sample mDNS records.
+The advertising agent should add an additional field to the TXT record:
```

Comment:
Nit: should = mandatory or optional? What happens if they don't?

Reply:
If an agent doesn't advertise the token, it will have a lower level of protection against outside parties attempting to authenticate to them (it will rely on a firewall/NAT to block incoming connections).

Maybe this makes sense in some closed environments or use cases.  I think there's a case to make it mandatory, but we'll need to consider future compatibility with cross-LAN authentication.

Will file an issue to consider the implications here.

Reply:
Filed Issue #185.